Today Extension/TodayExtensionCell.swift

```diff
@@ -0,0 +1,215 @@
+                make.top.equalTo(verticalMargin)
+                make.leading.equalTo(leftMargin)
+                make.height.equalTo(busIcon.intrinsicContentSize.height)
+                make.width.equalTo(busIcon.intrinsicContentSize.width)
+            }
+        }
+
+        destinationLabel.snp.makeConstraints { (make) in
+            make.top.equalTo(departureLabel.snp.bottom).offset(2)
+            make.leading.equalTo(departureLabel)
+            make.trailing.lessThanOrEqualToSuperview().inset(rightMargin)
+            make.height.equalTo(destinationLabel.intrinsicContentSize.height)
+        }
+
+        if showLiveElements {
+            liveLabel.snp.makeConstraints { (make) in
+                make.bottom.equalToSuperview().inset(verticalMargin)
+                make.leading.equalTo(departureLabel)
+                make.height.equalTo(liveLabel.intrinsicContentSize.height)
+                make.width.equalTo(liveLabel.intrinsicContentSize.width)
+            }
+
+            liveIndicatorView.snp.makeConstraints { (make) in
+                make.centerY.equalTo(liveLabel.snp.centerY)
+                make.leading.equalTo(liveLabel.snp.trailing).offset(8)
+                make.trailing.lessThanOrEqualTo(rightMargin)
+                make.width.equalTo(liveIndicatorView.intrinsicContentSize.width)
+            }
+        }
+    }
+
+    func setUpCell(route: Route?, destination: String) {
+        destinationName = destination
+        if let route = route {
+            self.route = route
+            if let departDirection = (route.directions.filter { $0.type == .depart }).first {
+                busDirection = departDirection
+                busIcon = BusIcon(type: .directionSmall, number: departDirection.routeNumber)
+                contentView.addSubview(busIcon!)
+
+                setUpDepartureLabel()
+                setUpDestinationLabel()
+                setUpLiveElements()
+            } else { // there is no bus to this destination (i.e. only walking)
+                setUpNoRoute()
+            }
+        } else { // no route at all
+            setUpNoRoute()
+        }
+    }
+
+    func setUpNoRoute() {
+        showLiveElements = false
+        let noRouteLabel = UILabel()
+        noRouteLabel.font = .getFont(.regular, size: 14.0)
+        noRouteLabel.textColor = Colors.primaryText
+        noRouteLabel.textAlignment = .center
+        noRouteLabel.lineBreakMode = NSLineBreakMode.byTruncatingTail
+        noRouteLabel.text = "No routes available to \(destinationName)."
+
+        contentView.addSubview(noRouteLabel)
+
+        noRouteLabel.snp.makeConstraints { (make) in
+            make.centerY.equalToSuperview()
+            make.centerX.equalToSuperview()
+            make.leading.equalToSuperview().inset(12.0)
+            make.trailing.equalToSuperview().inset(12.0)
+        }
+    }
+
+    func setUpDepartureLabel() {
+        departureLabel.lineBreakMode = NSLineBreakMode.byTruncatingTail
```

Comment:
Nitpicks: Remove Class name, move client-facing strings to Constants.swift

Reply:
I disagree here. I think "To" does not belong in Constants file. It is too much abstraction

Reply:
What about “No routes available”?